CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,44 @@
 
 import Foundation
 
-print("Hello, World!")
+// 로또 번호 생성
+
+func generateUniqueRandomNumbers() -> Set<Int> {
+    var uniqueNumbers = Set<Int>()
+    
+    while uniqueNumbers.count < 6 {
+        let randomNumber = Int.random(in: 1...45)
+        uniqueNumbers.insert(randomNumber)
+    }
+    
+    return uniqueNumbers
+}
+
+// 내가 고른 번호와 생성된 로또 번호 비교
+
+func compareLottoNumbers(_ choiceLottoNumbers: [Int]) {
+    if choiceLottoNumbers.count != 6 {
+        print("6개의 숫자를 선택해주세요.")
+        return
+    }
+    
+    if choiceLottoNumbers.contains(where: { $0 < 1 || $0 > 45 }) {
+        print("1부터 45사이의 숫자를 선택해주세요.")
+        return
+    }
+    
+    let myLottoNumbers: Set<Int> = Set(choiceLottoNumbers)
+    let generateLottoNumbers : Set<Int> = generateUniqueRandomNumbers()
```

Comment:
```suggestion
    let myLottoNumbers: Set<Int> = Set(choiceLottoNumbers)
    let generateLottoNumbers: Set<Int> = generateUniqueRandomNumbers()
```

추가적으로 generateLottoNumber라는 네이밍은 어떤 행위를 하는 느낌이 들어서 변수명에 어울리지 않는 것 같습니다 🤔

그리고 변수를 선언할 때 type annotation, type inference를 사용하는 히로님만의 기준이 있나요??

Reply:
type inference는 확실한 초기값이 있을 때 사용하고, 
특정한 초기값이 없을 경우에는 type annotation을 사용합니다